### PR TITLE
[preview] move to elastic brand

### PR DESCRIPTION
**Pull request by ph**
Mass update preview changes for https://github.com/elastic/logstash/issues/2832

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2012-2015 Elasticsearch <http://www.elasticsearch.org>
```
Comment (wiibaa) [inline LICENSE]: @ph shouldn't the company name in the copyright become "Elastic" too??

Comment (ph) [inline LICENSE]: The copyright is still associated with Elasticsearch not Elastic

Comment (wiibaa) [inline LICENSE]: Legal stuff is really not my cup of tea, better go for a :beer:

Comment (ph) [inline LICENSE]: Same here! not really into :necktie:

File: LICENSE [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
+Copyright (c) 2012–2015 Elasticsearch <http://www.elastic.co>
```
Comment (untergeek) [inline LICENSE]: Please ensure the use of an endash (no, I am not making this up) between 2012 and 2015:

2012–2015 

is proper.  Don't use an emdash either, which is too long: 2012—2015.  A PR came in to Elasticsearch for this to be corrected.  I'm too lazy/busy to look it up.

Comment (ph) [inline LICENSE]: Yes, I have looked up the commit :)

Comment (ph) [inline LICENSE]: https://github.com/elastic/logstash/commit/2123dc2ad1cce9c0c2cf0a9453d8443c621bd869

Comment (untergeek) [inline LICENSE]: Hah.  It's a Logstash PR.  Well, I knew I'd seen one.

**Comment by suyograo** [comment]
LGTM

File: LICENSE (hunk 1 above)
Comment (suyograo) [inline LICENSE]: yep copyright is still Elasticsearch

**Comment by jsvd** [comment]
fixed by https://github.com/logstash-plugins/logstash-output-statsd/commit/06e3941b184dc23311c131a316a4956f85c1dcc9

**Pull request closed without merging**